docs/develop/app-structure.md

```diff
@@ -68,29 +152,7 @@ On disk it might look like this:
         └── user.go             // user service code
 ```
 
-## Define components with sub-packages
-
-Within a service, it's possible to have multiple sub-packages. This is a good way to define components, should you wish to do that.
-
-To create sub-packages, you create sub-directories within a service package (which, again, is a regular Go package).
-
-Note that only service packages can define APIs, and sub-packages within services cannot themselves define APIs.
-You can however define an API in a service package that calls a function within a sub-package.
-
-On disk it might look like this:
-
-```
-/my-app
-├── encore.app                       // ... and other top-level project files
-│
-├── hello                            // hello service (a Go package)
-│   ├── migrations                   // hello service db migrations (directory)
-│   │   └── 1_create_table.up.sql    // hello service db migration
-│   ├── foo                          // sub-package foo (directory)
-│   │   └── foo.go                   // foo code (cannot define APIs)
-│   ├── hello.go                     // hello service code
-│   └── hello_test.go                // tests for hello service
-│
-└── world                            // world service (a Go package)
-    └── world.go                     // world service code
-```
\ No newline at end of file
+The only refactoring needed to separate a service application into systems is to move services into their respective
+subfolders. This greatly simplifies the application around the specific concerns of each system. For encore, what
+matters are the packages containing services. The division in systems or subsystems will not change the endpoint and
+architecture of your application
```

Comment:
```suggestion
The only refactoring needed to divide an existing Encore application into systems is to move services into their respective
subfolders. This is a simple way to separate the specific concerns of each system. What
matters for Encore are the packages containing services, and the division in systems or subsystems will not change the endpoints or
architecture of your application.
```

Reply:
"Service application" seemed a bit vague. So tweaked it to be a bit more use case centric, i.e. go from encore app without systems to one with systems.